doc/source/new-project-guide/level-3-tutorial.rst

```diff
@@ -68,12 +68,14 @@ If you would like to add more projects, you can create a new folder under the ro
 File descriptions
 ^^^^^^^^^^^^^^^^^^
 
-Now that the folder structure is created, let's go through the files and folders in the project:
+Now that the folder structure is created, let's go through the files and folders in the project.
+
+**Disclaimer:** The Python files (``.py``) and project structure shown here are example templates. You can **change file names, add files, edit contents, or replace the examples entirely** when creating your own project. These examples are intended to demonstrate key ideas like code reuse and organization.
 
 - ``README.md`` is a markdown file for project documentation. Typically, it includes a project description, installation instructions, and usage examples. You can edit this file to add your own project information.
 
 
-- ``shared_functions.py`` is where you define reused code. It includes the following example function to help demonstrate the concept of shared functions. In your project, this is where you would add the code you want to reuse:
+- ``shared_functions.py`` is where you define reused code. It includes the following example function to help demonstrate the concept of shared functions. In your project, this is where you would add the code you want to reuse across different projects.
```

Comment:
"...is the function where you define...."     

"...code.  It can have any name and you can have more than one file with different names containing your reusable code.  In the scikit-package template we populated with following example function...."